src/main/resources/static/css/main.css

```diff
@@ -1,28 +1,32 @@
-body{
-	padding-top:50px;
+body {
+	padding-top: 60px;
```

Comment:
Bootstrap 4 seems to favor rems as units for margins and paddings. You might experiment and see how much work it would be to convert. I'm guessing 50px ~ 1.5rem.